packages/storefront-lwc/src/modules/commerce/data/basket.js

```diff
@@ -7,236 +7,119 @@
 /**
  * A basket service to add to basket, load basket and blast off events
  */
-import { apiClient } from '../api/client';
 import gql from 'graphql-tag';
 
-class Basket {
-    basket = {};
-
-    listeners = [];
-
-    isBasketLoaded = false;
-
-    getBasketAttributes = `
-        basketId
-        customerId
-        getBasketMessage
-        totalProductsQuantity
-        shipmentId
-        shipmentTotal
-        selectedShippingMethodId
-        products {
-            productId
-            itemId
-            quantity
-            productName
-            price
-            imageURL
-            inventory {
-                ats
-                backorderable
-                id
-                orderable
-                preorderable
-                stockLevel
-            }
-            itemTotalAfterDiscount
-            itemTotalNonAdjusted
-            variationAttributes {
-                id
-                name
-                selectedValue {
-                    name
+const getBasketAttributes = `basketId
```

Comment:
In the case of product search and product details we removed the file in the data directory and add the queries in a new file (gqlQueries.js) inside the respective directories. We should consider doing the same to be consistent

Reply:
I agree and I plan to do that once I have received enough code comments. i had added that as a task to check with the team. Will talk in scrum